Update Teleport Cloud FAQ

wadells opened the pull request:
These topics come up frequently in our internal support channel.
    
Contributes to https://github.com/gravitational/cloud/issues/1297

* 2e4c6615561d9b3efab03b777e73cca49c6619ed has the notable changes. 
* 43349248400bea28f895660b9a7b02887d8c6167 is cleanup.

## Testing Done
Checked that the eventhandler link resolved correctly at:

https://github.com/gravitational/teleport/blob/walt/cloud-faq/docs/pages/cloud/faq.mdx

wadells committed:
Add info about downtime subscription and fluentd

These topics come up frequently in #dev-cloud-support.

Contributes to https://github.com/gravitational/cloud/issues/1297

wadells committed:
Formatting clean up

ulysseskan commented on docs/pages/cloud/faq.mdx:
```diff
@@ -150,6 +154,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my companies internal SEIM?
```
I think this should be `company's`.  And probably `SIEM` should be spelled out too along with the acronym.

wadells replied:
Changed both of these in f0a79a209cb3678b0de09d35986ae2ce740001d.  Thanks!

stevenGravy reviewed (commented):
For the question

## How can I access the tctl admin tool?

Can we add the link to the downloads page in cloud section?

wadells committed:
Add "where do I download tctl" to the FAQ

This comes up frequently.

wadells committed:
Fix a typo and clarify SEIM

wadells committed:
squash -- Clarify wording around downloads

wadells commented:
> Can we add the link to the downloads page in cloud section?

Fixed in [b82410b](https://github.com/gravitational/teleport/pull/12496/commits/b82410b3bf9f79b629953afbf26a27704da359be)

stevenGravy reviewed (approved):
lgtm

wadells committed:
Clarify Teleport Cloud data retention

This came up as a request while probing Customer Success for frequently
asked questions.

webvictim commented on docs/pages/cloud/faq.mdx:
```diff
@@ -150,6 +158,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my company's internal Security Event and Incident Managment (SEIM)?
```
I traditionally see "SIEM" used more than "SEIM", but maybe I'm behind the times?

wadells replied:
Good catch. Changed in [0e596c3](https://github.com/gravitational/teleport/pull/12496/commits/0e596c3bad29e200e8a27be85236450f7babebec).

wadells committed:
Update SEIM -> SEIM

ptgott commented on docs/pages/cloud/architecture.mdx:
```diff
@@ -21,11 +21,14 @@ Teleport Cloud Proxy does not terminate SSH sessions when using OpenSSH and `tsh
+When data retention policies are introduced, customers will be contacted and
+able to specify their preferred data retention schedules.
+
+Customers whose subscriptions lapse will have all session recordings, audit, and
```
```suggestion
Customers whose subscriptions lapse will have all session recordings, audit logs, and
```

ptgott commented on docs/pages/cloud/architecture.mdx:
```diff
@@ -21,11 +21,14 @@ Teleport Cloud Proxy does not terminate SSH sessions when using OpenSSH and `tsh
+able to specify their preferred data retention schedules.
+
+Customers whose subscriptions lapse will have all session recordings, audit, and
+cluster state deleted between 7 and 30 days of the lapse.
```
```suggestion
cluster state deleted between 7 and 30 days after the lapse.
```

ptgott commented on docs/pages/cloud/faq.mdx:
```diff
@@ -34,6 +38,10 @@ We use AWS-managed keys. Currently there is no option to provide your own key.
 
+## How long will Teleport Cloud retain my data?
+
+See our documentation on [data retention](./architecture.mdx#data-retention)
```
```suggestion
See our documentation on [data retention](./architecture.mdx#data-retention).
```

ptgott commented on docs/pages/cloud/faq.mdx:
```diff
@@ -43,14 +51,15 @@ There is no need to open any ports on your infrastructure for inbound traffic.
 ## How can I access the tctl admin tool?
 
-We have made changes to allow you to log into your cluster using `tsh`, then use `tctl` remotely:
+Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the enterprise version of `tctl` and `tsh`.
```
```suggestion
Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the Enterprise version of `tctl`.
```

My understanding is that there is no separate Enterprise version of `tsh` ([source](https://github.com/gravitational/teleport/pull/10920#discussion_r848898440)).

ptgott commented on docs/pages/cloud/faq.mdx:
```diff
@@ -43,14 +51,15 @@ There is no need to open any ports on your infrastructure for inbound traffic.
-We have made changes to allow you to log into your cluster using `tsh`, then use `tctl` remotely:
+Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the enterprise version of `tctl` and `tsh`.
+
+After downloading the tools, first log into your cluster using `tsh`, then use `tctl` remotely:
```
```suggestion
After downloading the tools, first log in to your cluster using `tsh`, then use `tctl` remotely:
```

ptgott commented on docs/pages/cloud/faq.mdx:
```diff
@@ -70,18 +79,18 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
+## Is an independent security audit available?
 
-Security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
+A security audit has been completed and is available [here](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/).
```
```suggestion
A security audit has been completed and is available at our [Audit Report 2021](https://goteleport.com/resources/audits/teleport-cloud-security-audit-2021/) page.
```
Added a suggestion to make this a bit clearer (https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking).

ptgott commented on docs/pages/cloud/faq.mdx:
```diff
@@ -70,18 +79,18 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 ## Where does Teleport Cloud run?
 
-Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS managed encryption keys. All data is currently hosted in us-west-2. 
+Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS managed encryption keys. All data is currently hosted in us-west-2.
```
```suggestion
Teleport Cloud is deployed within AWS, using S3 and DynamoDB for storage with AWS-managed encryption keys. All data is currently hosted in us-west-2.
```

ptgott reviewed (approved):
Approved with some minor suggestions

wadells committed:
Apply suggestions from code review

Co-authored-by: Paul Gottschling <paul.gottschling@goteleport.com>
